### DISCUSSION TITLE
EDIT TEXT

### DISCUSSION THREAD

**Discussion by AtizaD** [Stirling-Tools/Stirling-PDF#1262] [Q&A]
Please how do i edit text in a pdf file or that feature is not yet added?

**Comment by mauriva** [comment]
From what I've been able to see after a few days of use, I'm afraid it is not currently possible to directly edit a pdf file.
This is a shame because it is a very useful feature.
At the moment the only open source tool that allows pdf file editing is Draw from LibreOffice (at least it is the only one I know of).
I hope that such functionality can be included in the future.

**Reply by Frooodle** [reply]
Correct no way currently

**Reply by AtizaD** [reply]
I think Adobe Acrobat does it better when it comes to editing text directly in pdf.
We hope  @Frooodle can add that feature to this project. Good luck guys

**Reply by isaaczhao** [reply]
60+ features without text editing, do a better job before marketing hypes, uninstalling requires reboot

**Reply by Frooodle** [reply]
It does have text editing, this was added 4-6 months ago  and we actually have a update coming this week with a improved version?

It's a tool on right called pdf text editing (alpha) 

We are releasing a new version in beta soon

**Comment by kiwinh0** [comment]
Try [PDFGear](https://www.pdfgear.com/) app. 
I hope that soon they will be able to offer us this option in Stirling-PDF

**Comment by wlrsx** [comment]
It's a pity that such a powerful PDF tool doesn't have this function.

**Reply by DJB2-GitHub** [reply]
More than a pity. This tool is offered as a complete toolbox but doesn't have this basic feature. I spent ages attempting to change text and only just found out vi this thread (after much on-line searching) it cannot.

**Reply by Frooodle** [reply]
Sorry you're disappointed and I know it's a highly wanted feature but I assure you it's not basic, it's one of the most complex features to implement

**Comment by fcgll520** [comment]
For me, I often use [LightPDF](https://lightpdf.com/) software to edit text in PDF files, and it's very simple and convenient.

**Comment by DJB2-GitHub** [comment]
Thanks Anthony,
I feel chastened, as I should have recognized and thanked you for your work and development on this excellent and expansive product.

Regards Darryl

**Comment by Frooodle** [comment]
Text editing was added!
